_raml/apps/carts/carts.raml

```diff
@@ -139,7 +130,6 @@ post:
     displayName: Shipping address
     put:
       description: Modifies the shipping address for a cart.
-      securedBy: [carts_write]
```

Comment:
Since all the cart resources do not need any scope, we have the following situation on detail pages in the doku:

> [...]
> Authorisation
> 
> OAuth 2.0 Token required with scopes: 
>
> URI parameters
> [...]

It could be a task for team tools to remove the *Authorisation* paragraph if no scope is needed